packages/navbrand/src/css/index.js

```diff
@@ -0,0 +1,51 @@
+import {
+  colorsBackgroundUtility,
+  layout
+} from '@pluralsight/ps-design-system-core'
+
+import { transparentize } from '@pluralsight/ps-design-system-util'
+
+export default {
+  '.psds-navbrand': {
```

Comment:
This element needs `border-radius: 2px` and height should always be 40px.

Reply:
done